common/beerocks/bwl/include/bwl/base_wlan_hal_types.h

```diff
@@ -46,17 +46,21 @@ enum class AntMode { Invalid = 0, ANT_1X1, ANT_2X2, ANT_3X3, ANT_4X4 };
 enum class WiFiChanBW { Invalid = 0, BW_20 = 20, BW_40 = 40, BW_80 = 80 };
 
 struct VAPElement {
+    /**
+     * Basic Service Set (i.e.: VAP name, e.g.: wlan0.0, wlan0.1, wlan0.2, ...).
+     */
+    std::string bss;
```

Comment:
This commit breaking the build because the compiler will mark `std::string bss` as an unused variable.

Reply:
I've compiled this commit with no error. 
I would probably get an error if `bss` was an unused local or global variable, but it's a member in a struct.